diff -Naur procps-ng-3.3.2.orig/skill.c procps-ng-3.3.2/skill.c
--- procps-ng-3.3.2.orig/skill.c	2012-01-24 11:41:53.000000000 -0500
+++ procps-ng-3.3.2/skill.c	2012-01-24 11:42:20.000000000 -0500
@@ -74,6 +74,11 @@
 	fprintf(stdout, PROCPS_NG_VERSION);
 }
 
+#ifndef HAVE_RPMATCH
+#define rpmatch(r) \
+ (*r == 'y' || *r == 'Y' ? 1 : *r == 'n' || *r == 'N' ? 0 : -1)
+#endif
+
 /* kill or nice a process */
 static void hurt_proc(int tty, int uid, int pid, const char *restrict const cmd,
 		      struct run_time_conf_t *run_time)
